bench/bench_test.go

```diff
@@ -77,7 +77,7 @@ func runBase64Benches(b *testing.B, store *wasm.Store) {
 }
 
 func runFibBenches(b *testing.B, store *wasm.Store) {
-	for _, num := range []int{5, 10, 20} {
+	for _, num := range []int{5, 10, 20, 30} {
```

Comment:
is this special? if not, ignore my comment.

Reply:
I just added this as the more number in fibonacci case means the more call heavy, and that's it. It's not so special!